.github/workflows/wss-scan.yml

```diff
@@ -0,0 +1,46 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 3 * * *'
```

Comment:
3:30 here is UTC time, are you sure you want to use this time which is 10:30 PST?

Reply:
Thanks for the info. Made the change as required to run at 3.30 AM PDT